content/blog/2022/09/2022-09-18-digital-ocean-sponsorship.adoc

```diff
@@ -0,0 +1,29 @@
+---
+layout: post
+title: "Digital Ocean and Jenkins partnership continues to grow!"
+tags:
+- jenkins
+- digitalocean
+- community
+author: kmartens27
+description: "DigitalOcean increases Jenkins sponsorship"
+opengraph:
+  image: /images/post-images/2022-09-18-digital-ocean-sponsorship/DO-horizontal.png
+---
+
+We are excited to announce that DigitalOcean and Jenkins' relationship continues to grow!
+Eternal thanks to DigitalOcean for providing such incredible support to the Jenkins project.
+image:/images/post-images/2022-09-18-digital-ocean-sponsorship/DO-horizontal.png[link="https://www.digitalocean.com"]
```

Comment:
I would put an empty line before the logo
```suggestion

We are excited to announce that DigitalOcean and Jenkins' relationship continues to grow!
Eternal thanks to DigitalOcean for providing such incredible support to the Jenkins project.

image:/images/post-images/2022-09-18-digital-ocean-sponsorship/DO-horizontal.png[link="https://www.digitalocean.com"]
```

or even put it at the top of the article:
```suggestion
image:/images/post-images/2022-09-18-digital-ocean-sponsorship/DO-horizontal.png[link="https://www.digitalocean.com"]

We are excited to announce that DigitalOcean and Jenkins' relationship continues to grow!
Eternal thanks to DigitalOcean for providing such incredible support to the Jenkins project.
```